kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -311,23 +310,26 @@ function showLearnChannel(store, channelId) {
     const popularPromise = ContentNodeResource.getCollection(popularPayload).fetch();
     const resumePromise = ContentNodeResource.getCollection(resumePayload).fetch();
     _updateChannelList(store);
-    ConditionalPromise.all([nextStepsPromise, popularPromise, resumePromise])
-      .only(checkSamePageId(store), ([nextSteps, popular, resume]) => {
+    ConditionalPromise.all([nextStepsPromise, popularPromise, resumePromise]).only(
+      samePageCheckGenerator(store),
+      ([nextSteps, popular, resume]) => {
         const pageState = { recommendations: { nextSteps: nextSteps.map(_contentState),
                                                popular: popular.map(_contentState),
                                                resume: resume.map(_contentState) } };
         store.dispatch('SET_PAGE_STATE', pageState);
         store.dispatch('CORE_SET_PAGE_LOADING', false);
         store.dispatch('CORE_SET_ERROR', null);
-      }, (error) => {
+      },
+      (error) => {
         store.dispatch('CORE_SET_ERROR', JSON.stringify(error, null, '\t'));
         store.dispatch('CORE_SET_PAGE_LOADING', false);
-      });
+      }
+    );
   })
-    .catch((error) => {
-      store.dispatch('CORE_SET_ERROR', JSON.stringify(error, null, '\t'));
-      store.dispatch('CORE_SET_PAGE_LOADING', false);
-    });
+  .catch((error) => {
```

Comment:
Should we lose this `catch` too, or is there an error we think likely to be thrown and that we want to display to the user?